pre-stop.d/02_dump_xivo_service_to_file.py

```diff
@@ -26,12 +26,12 @@
 EXCLUDE_SERVICES = [
     'asterisk',
     'wazo-auth',
+    'wazo-agentd-cli',
     'wazo-call-logd',
     'wazo-dird-xivo-backend',
     'wazo-plugind',
     'wazo-plugind-cli',
     'wazo-upgrade',
-    'xivo-agentd-cli',
```

Comment:
it is a pre-stop script, so the script is ran before the migration

Reply:
I'm supposed to keep both then ?

Reply:
No, this script is only executed on version < 18.13 ... So you have nothing to do